client/src/components/MobileItemList.vue

```diff
@@ -72,7 +72,7 @@ const props = defineProps<{
   thirdLabel?: string | number
 }>();
 
-const emit = defineEmits<{
+defineEmits<{
```

Comment:
did one of the rule change that?

Reply:
Nah, it was just an unused variable so I thought I might as well get rid of it. Still on less warning for the linter.